.github/workflows/deployAfterModelRegistered.yml

```diff
@@ -0,0 +1,100 @@
+name: aml-deploy
```

Comment:
please change this name property in workflows to represent what they do.. it is also used to track from the list

Reply:
will update

Reply:
updated